app/src/main/res/values/strings.xml

```diff
@@ -242,41 +242,43 @@
+    <string name="login_forgot_password">Forgot your password?</string>
 
-    <string name="login_wrong_username_or_password">Couldn\'t authenticate (probably wrong username or password).\nHave you registered?</string>
+    <string name="login_configuration_detection">Setup detection</string>
+    <string name="login_querying_server">Please wait, querying server…</string>
+    <string name="login_wrong_username_or_password">Could not log in (probably wrong username or password).\nHave you registered?</string>
     <string name="login_view_logs">View logs</string>
 
     <string name="setting_up_encryption">Setting up encryption</string>
     <string name="setting_up_encryption_content">Please wait, setting up encryption…</string>
 
-    <string name="account_creation_failed">Account creation failed</string>
+    <string name="account_creation_failed">Could not create account</string>
     <string name="wrong_encryption_password">Wrong encryption password</string>
-    <string name="wrong_encryption_password_content">Got an integrity error while accessing your account, which most likely means you put in the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
+    <string name="wrong_encryption_password_content">Could not access your account. Most likely you entered the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
 
     <string name="signup_title">Enter Signup Details</string>
     <string name="signup_password_restrictions">Password should be at least 8 characters long</string>
 
     <!-- ChangeEncryptionPasswordActivity -->
     <string name="change_encryption_password_title">Change Encryption Password</string>
-    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe your encryption password has been compromised. Contact support instead.</string>
+    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe your encryption password is in the wrong hands. Contact support instead.</string>
     <string name="change_encryption_password_new_password">New encryption Password</string>
     <string name="change_encryption_password_are_you_sure">Are you sure you would like to continue? The process can not be stopped half way or be undone, and may take a while to complete.</string>
     <string name="change_encryption_password_success_title">Encryption Password Change</string>
-    <string name="change_encryption_password_success_body">Encryption password has been successfully changed!</string>
+    <string name="change_encryption_password_success_body">Use the new encryption password to unlock your tasks next time</string>
```

Comment:
This change doesn't make sense.

Reply:
It isn't otherwise clear what an encryption password is/does.(?)
That's the idea anyhow.

Reply:
It sounds to me like you are not very familiar with the EteSync app, have you used it before? Because it doesn't just do tasks... Also, the button that leads to it is labelled "change encryption password". I think the previous string makes a lot of sense.

Reply:
True. I was trying to make it functional. What is the most generic way to describe what EteSync does?
The word "successfully" doesn't do anything, and the exclamation mark is unwarranted. People will translate "!" into languages where it is even worse for the user than in English.

Reply:
I'm not saying what was there is great, just that the new translation makes much less sense than the previous one.
"Password successfully changed" is also fine by me.
The successfully doesn't add meaning, I agree, though it makes it visually better. "Password changed" is too short.

Reply:
Changed to "data"

Reply:
I think it's overly complex. It's the same way we have "Login successful" and not "Login successful, you can now use the app to access your data". It's unnecessary and obvious information.

Reply:
There is no "Login successful" though…? Nor is it different from "Login successful". All the "successful" is gone.
Here is how "Encryption password has been successfully changed!" breaks down.
1. What is encryption?
2. What is an encryption password
3. What does an encryption password do?
4. Why "successfully", is there some other not quite successful state?
5. Why the exclamation, it sounds like being able to change the password is an unlikely event.

In 13 chars more, the user knows
1. That there is an encryption password
2. It is new
3. It is used to unlock the tasks
4. Deductive, without it, unlocking the tasks isn't possible, and the user should record the password now, for fear of not being able to.

I think the only improvement here is a "you must", instead, and a personification.

Reply:
At this point they already know the answer to all of your questions because they are *changing* something they have set before. They've been educated (or not) about these concepts when they first set the encryption password when they first signed up.

As for 4/5:
4. "X finished successfully" is a common pattern. It's because you want to indicate success but you really have nothing to say.
5. I agree, get rid of it.

> In 13 chars more, the user knows ...

As I said, they should already know there's an encryption password, they set it. It's not new, they changed something. Take a look at the UI in question, is very obvious that it's new. You are literally prompted for the old and new passwords. It's not just used to unlock tasks, it's used for all of your data. Now with the new EteSync notes it's also for that. You really can't know from this app what it would be used for. As I said previously, this is *not* tasks specific.

As I said elsewhere, please use the app and see where it's used.

> Possibly "Remember your new encryption password if you want to unlock your data next time" is better.

I think we already added a notice for that in the dialog itself (or it's in my todo), as you want people to remember it *before* the change applies.

Again, I'm sorry, but I just disagree with the whole premise of your argument. It's a really standard thing, change your password in literally every app out there and you'll get "password successfully changed".

Reply:
Is this still in the process of setting things up, or is there a lot of time in between? There is a disconnect of possibly years(?) I still don't think it hurts to reiterate. The idea of having two passwords is unique, and it isn't out of the ordinary people forget the difference between login and unencryption passwords for e-mail.

"password successfully changed" is only found in one other project https://hosted.weblate.org/search/?offset=1&q=%22password+successfully+changed%22&sort_by=-priority%2Cposition&checksum=

Windows does not use "successfully changed" http://www.fullerton.edu/it/_resources/images/password/pc_desktop/changesuccessful.png nor does Android to my knowledge. I tried to search for it and only got software I had never heard about.

What is being encrypted other than tasks?
How about "Encryption password changed". It isn't good, but it isn't bad on the assumption people know what is being encrypted already.

Reply:
Sorry, I didn't mean the actual string "Successfully changed", I meant the same kind of string (compared to what you were suggesting). The windows one is a good string and I'm happy to use that.

Starting from EteSync v2 there are no longer two passwords, just one. Maybe that's where the confusion stems from. It should now be "The password has been changed". I'm happy with that.

Reply:
OK, changing.